tests/console/check_package_version.pm

```diff
@@ -61,18 +40,18 @@ sub cmp_version {
 }
 
 sub cmp_packages {
-    my ($pcks, $pckv, $jsc) = @_;
+    my ($pcks, $pckv) = @_;
     record_info($pcks, "$pcks version check after migration");
     my $output = script_output("zypper se -xs $pcks | grep -w $pcks | head -1 | awk -F '|' '{print \$4}'", 100, proceed_on_failure => 1);
     my $out    = '';
     for my $line (split(/\r?\n/, $output)) {
         if (trim($line) =~ m/^\d+\.\d+(\.\d+)?/) {
             $out = $line;
-            record_soft_failure("$jsc, The $pcks version is $out, but request is $pckv") if (!cmp_version($pckv, $out));
+            record_soft_failure("The $pcks version is $out, but request is $pckv") if (!cmp_version($pckv, $out));
```

Comment:
I think you need update the com_packages to compare little version which is not stable as '0'.

It means you can skip the compare for little version if the expected version is 13.0.0. A quick thinking, change the '13.0.0' to '13' for example and compare corresponding bits is ok.

Reply:
I don't think it is necessary to skip the little version, it meet the checking requirement.